build.gradle

```diff
@@ -87,7 +82,7 @@ task integrationTest(type: Test) {
 
 task printMessageAboutRunningServer {
     doLast {
-        println "The server is now running at http://localhost:${testServerHttpPort}/${warContext}"
+        println "The server is now running at http://localhost:${testServerHttpPort}/jaxrs-async"
```

Comment:
The warContext is set above as `warContext = 'async-jaxrs'` -- async-jaxrs vs jaxrs-async. Which is it?   And since you are still using the warContext variable in line 85, I hope that the variable is correct.  Switch this back to using ${warContext}